sentry-logback/src/test/kotlin/io/sentry/logback/SentryAppenderTest.kt

```diff
@@ -82,17 +89,25 @@ class SentryAppenderTest {
 
     @Test
     fun `does not initialize Sentry if Sentry is already enabled`() {
```

Comment:
afaict this test didn't actually test what it says it does. It used to just initialize Sentry via the log appender then reinitialize using `Sentry.init` and it asserted that the env from re-init was set.